### DISCUSSION TITLE
We're looking forward to translation contributors!

### DISCUSSION THREAD

**Discussion by lucydodo** [sqlitebrowser/sqlitebrowser#3736] [General]
Hello Community,

To make things easier for our translators, lets try using a wiki page to co-ordinate who's doing what:  
https://github.com/sqlitebrowser/sqlitebrowser/wiki/Translation-Contributor-List

That has a list of GitHub users and languages we'll notify before new releases.

As its a wiki page people can directly add/update/remove themselves as desired from translator duties. :) By listing your GitHub username on the wiki page, you can receive translation requests for new releases and review requests for translation PRs.
Please feel welcome to update it over time with your details, or remove them as needed.

Thank you to all of our translation volunteers, as they're directly responsible for spreading DB4S usage to many native languages. :)

**Comment by acid-jack** [comment]
Hello. Here is updated Ukrainian translation.
[sqlb_uk_UA.ts.gz](https://github.com/user-attachments/files/22461520/sqlb_uk_UA.ts.gz)

I'm too lazy to fork the project and make PR :)
Maybe you'll find some other methods to update translations.
